src/core/client/stream/mutations/DeleteCommentReactionMutation.ts

```diff
@@ -0,0 +1,52 @@
+import { graphql } from "react-relay";
+import { Environment } from "relay-runtime";
+
+import {
+  commitMutationPromiseNormalized,
+  createMutationContainer,
+} from "talk-framework/lib/relay";
+
+import { DeleteCommentReactionMutation as MutationTypes } from "talk-stream/__generated__/DeleteCommentReactionMutation.graphql";
+import { CreateCommentReactionInput } from "./CreateCommentReactionMutation";
+
+const mutation = graphql`
+  mutation DeleteCommentReactionMutation($input: CreateCommentReactionInput!) {
+    deleteCommentReaction(input: $input) {
+      comment {
+        ...ReactionButtonContainer_comment
+      }
+      clientMutationId
+    }
+  }
+`;
+
+const clientMutationId = 0;
+
+function commit(environment: Environment, input: CreateCommentReactionInput) {
+  return commitMutationPromiseNormalized<MutationTypes>(environment, {
+    mutation,
+    variables: {
+      input: {
+        ...input,
+        clientMutationId: clientMutationId.toString(),
+      },
+    },
+    optimisticResponse: {
+      deleteCommentReaction: {
+        comment: {
+          id: input.commentID,
+        },
+        clientMutationId: clientMutationId.toString(),
```

Comment:
actionCount is missing for the same reason as stated in the previous review.

Reply:
Done!